Dockerfile

```diff
@@ -15,13 +15,17 @@ COPY /tailwind.config.js /code
 RUN poetry install
 RUN npm install
 
-FROM base as production
+FROM base as test
 
-COPY /todo_app /code/todo_app
-RUN npx tailwindcss -i ./todo_app/tailwind.css -o ./todo_app/static/css/index.css
-
-ENTRYPOINT poetry run gunicorn -w=4 --bind 0.0.0.0 "todo_app.app:create_app()"
+COPY / /code
```

Comment:
(Random comment, not recommendation)
I always find `COPY /` much less intuitive than `COPY .` even though they're functionally the same - the latter allows me to not think very much about the concept of the build-context ("it's just what folder I'm in") whereas `COPY /` forces me to remember that actually the build context is first identified and _then_ used to copy files in
On reflection, not sure if I prefer the syntax that is more beginner friendly, or the one that implies more about reality. Interesting philosophical question!

(Also, the fact I'm musing about this is an interesting statement about the relative frequency with which I see the two approaches - `COPY .` seems much more common, but maybe because it's what Alex and I default to!)

Reply:
My entire reason for using "/" when possible is because I often skim over "."s and get confused what a command is doing 😂